app/mailers/event_mailer.rb

```diff
@@ -0,0 +1,11 @@
+# :reek:InstanceVariableAssumption
+class EventMailer < ActionMailer::Base
+  default from: 'nicher@example.com'
+
+  def event_deleted_email
+    @email = params[:email]
+    @event = params[:event]
+    @subscribers_email = params[:subscribers].pluck(:email)
+    mail(to: @subscribers_email, subject: 'Notify event deleted')
```

Comment:
It will be good to have event name and the reason of notification(cancellation of event) in email subject
I'd name it "#{@event.name} is cancelled"